src/redux/reducers/main.ts

```diff
@@ -202,6 +209,9 @@ export const mainSlice = createSlice({
   name: 'main',
   initialState: initialState.main,
   reducers: {
+    setMainProcessEnv: (state: Draft<AppState>, action: PayloadAction<any>) => {
+      mainProcessEnv = action.payload;
```

Comment:
this is a bit weird, should we live it like this for now?

the alternative would be to create an ipc event and send that from main process instead of dispatching this action

Reply:
that's what I had initially - i preferred this approach - but maybe an IPC event is easier to understand!?